bin/run_swiftlint

```diff
@@ -2,11 +2,24 @@
 
 echo "--- :swift: Running SwiftLint"
 
-SWIFTLINT_VERSION=$(<.swiftlint.yml awk '/^swiftlint_version: / {print $2}')
-SWIFTLINT_CMD="docker run --rm -v $PWD:/workspace -w /workspace ghcr.io/realm/swiftlint:$SWIFTLINT_VERSION swiftlint"
+SWIFTLINT_ARGUMENTS=(--quiet --reporter relative-path)
+
+if [ "$1" == "--strict" ] || [ "$1" == "--lenient" ]; then
+  SWIFTLINT_ARGUMENTS+=("$1")
+  shift
+fi
```

Comment:
Only drawback of this code is that once it consumed `$1` it doesn't continue searching for subsequent arguments.
[For example, in our other commands](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/blob/trunk/bin/validate_podspec#L20), instead of a single `if`, we do a `while` loop to parse all potential flags, not just the first one.

Granted, in the specific case of this `run_swiftlint` command, since we only expect one argument at most (either `--strict` or `--lenient` or neigher of them) and never a second one… this is not super impactful for the moment. Except that:
 - If someone decides to run `run_swiftlint --lenient --strict` (probably not literally, but because one of their `.buildkite/commands/myscript.sh` is not written as well as it should for example?), then this logic will see the `--lenient` and totally ignore the `--strict`
 - This is not super future-proof, as if one day we want to support more arguments we'll have to remember to transform that into a `while` loop as opposed to being tempted to do some copy/paste of the current parsing code and risk imposing a constraint on the additional arguments' order.

Reply:
+1 for consistency and extensibility. Update on 3689d57e4ff3cc5795fe5f25eb417295b8c6b329.